helm/idol-library/templates/aciserver/_container.tpl

```diff
@@ -66,9 +66,17 @@ volumeMounts:
   subPath: versionkey.dat
   readOnly: true
 {{- end }}
+{{- $idolComponentCfgExists := false }}
+{{- range $env }}
+  {{- if eq .name "IDOL_COMPONENT_CFG" }}
+    {{- $idolComponentCfgExists = true }}
+  {{- end }}
+{{- end }}
 env:
+{{- if not $idolComponentCfgExists }}
 - name: IDOL_COMPONENT_CFG
   value: {{ printf "/etc/config/idol/%s.cfg" (trimPrefix "idol-" $component.name) }}
+{{- end }}
 {{- range $env }}
```

Comment:
I think this isn't strictly necessary. Various sources suggest that the 'last' value will take priority, so any instance of `IDOL_COMPONENT_CFG` in `$env` should override the first value provided here.

Reply:
Yeah it does, the only reason for this change was to prevent a warning like follows:

> W1021 08:51:46.589346 2357339 warnings.go:70] spec.template.spec.containers[0].env[1].name: duplicate name "IDOL_COMPONENT_CFG"

Reply:
Ah, okay.